.github/actions/setup/action.yml

```diff
@@ -22,51 +22,136 @@ inputs:
+        }
+
+        # Get headers and shared libraries in place
+        populate_cuda_path cuda_nvcc
+        populate_cuda_path cuda_cudart
+        populate_cuda_path cuda_nvrtc
+        populate_cuda_path cuda_profiler_api
+        ls -l $CUDA_PATH
 
-        TARGET_PLATFORM='linux-64'
-        if [[ "${{ inputs.host-platform }}" == "linux-aarch64" ]]; then
+        # Note: the headers will be copied into the cibuildwheel manylinux container,
+        # so setting the CUDA_PATH env var here is meaningless.
+
+    - name: Set environment variables
+      shell: bash --noprofile --norc -xeuo pipefail {0}
+      run: |
+        # TODO: just align host-platform names with TARGET_PLATFORM...
+        if [[ "${{ inputs.host-platform }}" == "linux-x64" ]]; then
+          TARGET_PLATFORM='linux-64'
+        elif [[ "${{ inputs.host-platform }}" == "linux-aarch64" ]]; then
           TARGET_PLATFORM='linux-aarch64'
+        elif [[ "${{ inputs.host-platform }}" == "win-x64" ]]; then
+          TARGET_PLATFORM='win-64'
         fi
 
-        BUILD_MODE="${{ inputs.build-mode }}"
-        BUILD_MODE_STR=""
-        [ -n "${BUILD_MODE}" ] && BUILD_MODE_STR="-${BUILD_MODE}"
+        if [[ "${{ inputs.host-platform }}" == linux* ]]; then
+          if [[ "${{ inputs.python-version }}" == "3.12" ]]; then
+            CIBW_BUILD='cp312-manylinux*'
+          elif [[ "${{ inputs.python-version }}" == "3.11" ]]; then
+            CIBW_BUILD='cp311-manylinux*'
+          elif [[ "${{ inputs.python-version }}" == "3.10" ]]; then
+            CIBW_BUILD='cp310-manylinux*'
+          elif [[ "${{ inputs.python-version }}" == "3.9" ]]; then
+            CIBW_BUILD='cp39-manylinux*'
+          fi
+          REPO_DIR=$(pwd)
+        elif [[ "${{ inputs.host-platform }}" == win* ]]; then
+          if [[ "${{ inputs.python-version }}" == "3.12" ]]; then
+            CIBW_BUILD='cp312-win_amd64'
+          elif [[ "${{ inputs.python-version }}" == "3.11" ]]; then
+            CIBW_BUILD='cp311-win_amd64'
+          elif [[ "${{ inputs.python-version }}" == "3.10" ]]; then
+            CIBW_BUILD='cp310-win_amd64'
+          elif [[ "${{ inputs.python-version }}" == "3.9" ]]; then
+           CIBW_BUILD='cp39-win_amd64'
+          fi
+          PWD=$(pwd)
+          REPO_DIR=$(cygpath -w $PWD)
+        fi
 
+        BUILD_MODE="${{ inputs.build-mode }}"
         if [[ ("${BUILD_MODE}" == "") || ("${BUILD_MODE}" == "release") ]]; then
           # We upload release versions in the default folder.
           PKG_DIR="${TARGET_PLATFORM}"
         else
           PKG_DIR="${BUILD_MODE}/${TARGET_PLATFORM}"
         fi
 
-        PYTHON_VERSION_FORMATTED=$(echo '${{ inputs.python-version }}' | tr -d '.')
-
-        echo "BINDINGS_ARTIFACT_NAME=${{ inputs.host-platform }}-${{ inputs.build-type }}-cuda_bindings-python${PYTHON_VERSION_FORMATTED}-${{ inputs.target-device }}${BUILD_MODE_STR}${WITH_TESTS_STR}-${{ github.sha }}" >> $GITHUB_ENV
```

Comment:
or could do it like this with the PYTHON_VERSION_FORMATTED var

Reply:
yup, done in commit 07b8d1d